code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -53,23 +53,23 @@
 			new /obj/item/card/emag(src) // 6TC
 			new /obj/item/clothing/gloves/color/yellow(src) // 0TC
 			new /obj/item/grenade/syndieminibomb(src) // 6TC
-			new /obj/item/grenade/clusterbuster/n2o(src) // 0TC
+			new /obj/item/grenade/clusterbuster/n2o(src) // 4TC
 			new /obj/item/storage/box/syndie_kit/space(src) // 4TC
 			new /obj/item/encryptionkey/syndicate(src) // 2TC
 			return
 
-		if("payday") // 33TC + four 0TC
+		if("payday") // 35TC + four 0TC
 			new /obj/item/gun/projectile/revolver(src) // 13TC
 			new /obj/item/ammo_box/a357(src) // 3TC
 			new /obj/item/ammo_box/a357(src) // 3TC
 			new /obj/item/card/emag(src) // 6TC
-			new /obj/item/grenade/plastic/c4(src) // 1TC
```

Comment:
Since you're removing the C4, which was meant to blow open the vault door, perhaps replace the emag with an access tuner? The access tuner is perfect to unbolt and open the reinforced vault door and it's the same TC value.

Reply:
Payday is more geared for a louder approach to objectives, and the emag is generally preferable for that as well as being able to open lockers. Access Tuner kinda feels like it belongs in a stealthier kit, possibly in it's own bundle with a camera bug or something.
I get that the access tuner is better for the vault door, but I'd rather the kit was more useful in general than being geared solely for one single objective, the drill is there more for the theme than anything.

Reply:
They now can't break into the vault using solely the kit. Couldn't before already though since they should have only gotten one C4 charge